x-pack/plugins/observability_solution/slo/server/services/get_slos_overview.ts

```diff
@@ -0,0 +1,152 @@
+import { typedSearch } from '../utils/queries';
+import { getElasticsearchQueryOrThrow } from './transform_generators';
+import { getListOfSummaryIndices } from './slo_settings';
+
+export class GetSLOsOverview {
+  constructor(
+    private soClient: SavedObjectsClientContract,
+    private esClient: ElasticsearchClient,
+    private spaceId: string,
+    private logger: Logger,
+    private rulesClient: RulesClientApi,
+    private racClient: AlertsClient
+  ) {}
+
+  public async execute(params: GetOverviewParams = {}): Promise<GetOverviewResponse> {
+    const { indices, settings } = await getListOfSummaryIndices(this.soClient, this.esClient);
+
+    const kqlQuery = params.kqlQuery ?? '';
+    const filters = params.filters ?? '';
+    let parsedFilters: any = {};
+    try {
+      parsedFilters = JSON.parse(filters);
+    } catch (e) {
+      this.logger.error(`Failed to parse filters: ${e.message}`);
+    }
+
+    const response = await typedSearch(this.esClient, {
+      index: indices,
+      size: 0,
+      query: {
+        bool: {
+          filter: [
+            { term: { spaceId: this.spaceId } },
+            getElasticsearchQueryOrThrow(kqlQuery),
+            ...(parsedFilters.filter ?? []),
+          ],
+          must_not: [...(parsedFilters.must_not ?? [])],
+        },
+      },
+      body: {
+        aggs: {
+          worst: {
+            top_hits: {
+              sort: {
+                errorBudgetRemaining: {
+                  order: 'asc',
+                },
+              },
+              _source: {
+                includes: ['sliValue', 'status', 'slo.id', 'slo.instanceId', 'slo.name'],
+              },
+              size: 1,
+            },
+          },
+          stale: {
+            filter: {
+              range: {
+                summaryUpdatedAt: {
+                  lt: `now-${settings.staleThresholdInHours}h`,
+                },
+              },
+            },
+          },
+          violated: {
+            filter: {
+              term: {
+                status: 'VIOLATED',
+              },
+            },
+          },
+          healthy: {
+            filter: {
+              term: {
+                status: 'HEALTHY',
+              },
+            },
+          },
+          degrading: {
+            filter: {
+              term: {
+                status: 'DEGRADING',
+              },
+            },
+          },
+          noData: {
+            filter: {
+              term: {
+                status: 'NO_DATA',
+              },
+            },
+          },
+        },
+      },
+    });
+
+    const [rules, alerts] = await Promise.all([
+      this.rulesClient.find({
+        options: {
+          search: 'alert.attributes.alertTypeId:("slo.rules.burnRate")',
+        },
+      }),
+
+      this.racClient.getAlertSummary({
+        featureIds: observabilityAlertFeatureIds,
+        gte: moment().subtract(24, 'hours').toISOString(),
+        lte: moment().toISOString(),
```

Comment:
Let's make sure we are using the same date range for the Active Alert badge displayed on the SLO pages. Otherwise we might report more active alerts here than we have in reality